gammapy/makers/map.py

```diff
@@ -412,4 +414,9 @@ def run(self, dataset, observation):
 
             kwargs["edisp"] = edisp
 
+        kwargs["livetime_map"] = (
+            kwargs["mask_safe"].reduce_over_axes(func=np.logical_or)
```

Comment:
I would leave the livetime map un-masked here...

Reply:
The livetime map for a single observation should take the mask into account

Reply:
For single observations we leave the quantities unmasked. The mask is only applied when stacking or calling `.to_masked()`. This has the huge advantage that the safe data range can be modified after data reduction if needed.

Reply:
Yes indeed, my bad. If the counts are there, then the livetime should also be.